### PR TITLE
Feature header style

### PR DESCRIPTION
**Pull request by CarlosAlbertoSantos**
Following the layaut of the login determined by the members, I created the header with its functionality and styles.

- Desktop;
- Desktop with two buttons;
- Mobile with two buttons;

desktop-button- login
desktop-button-registration
desktop-buttons
mobile-button-login
mobile-button-registration
mobile-buttons

### CODE REVIEW

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  <section class="container flex">
+    <div class="logo">
+      <a href="#">Word Cup Bolão</a>
+    </div>
+
+    <% if login_page? %>
+      <%= link_to(
+          "Registrar",
+          new_registration_path(:user),
+          class:"button button--transparent"
```
Comment (frenangomes) [inline app/views/layouts/_header.html.erb]: 💅 use space between key and value, like this: 
```
...
class: 'button button--transparent'
```

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          ) %>
+
+    <% elsif registration_page? %>
+      <%= link_to(
+          "LOGIN",
+          new_session_path(:user),
+          class:"button button--transparent"
```
Comment (frenangomes) [inline app/views/layouts/_header.html.erb]: the same ☝️

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+          ) %>
+
+    <% else %>
+      <div>
+        <%= link_to(
+          "LOGIN",
+          new_session_path(:user),
+          class:"link white"
```
Comment (frenangomes) [inline app/views/layouts/_header.html.erb]: ☝️

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+          ) %>
+        <%= link_to(
+          "Registrar",
+           new_registration_path(:user),
+          class:"button button--transparent"
```
Comment (frenangomes) [inline app/views/layouts/_header.html.erb]: ☝️

File: spec/helpers/application_helper_spec.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe ApplicationHelper do
```
Comment (frenangomes) [inline spec/helpers/application_helper_spec.rb]: ❤️

File: spec/helpers/application_helper_spec.rb [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+  describe '.registration_page?' do
+    it 'must be registration page' do
+      helper.request.path = '/users/sign_up'
+
+      expect(helper.registration_page?).to be_truthy
+    end
+  end
+end
+
```
Comment (frenangomes) [inline spec/helpers/application_helper_spec.rb]: ✂️

**Comment by frenangomes** [comment]
What do you think about putting the other layouts in the descriptions?

For example, you can add the following layouts:
- Desktop;
- Desktop with two buttons;
- Mobile with two buttons;

File: app/views/layouts/_header.html.erb (hunk 1 above)
Comment (mayconbalves) [inline app/views/layouts/_header.html.erb]: why ?

File: app/views/layouts/_header.html.erb (hunk 2 above)
Comment (mayconbalves) [inline app/views/layouts/_header.html.erb]: again ?

**Comment by CarlosAlbertoSantos** [comment]
Can be! 🤔

File: app/views/layouts/_header.html.erb (hunk 1 above)
Comment (CarlosAlbertoSantos) [inline app/views/layouts/_header.html.erb]: We determined a pattern of single quotes at the beginning of the Maycon project, I had forgotten.

**Pull request merged**